site/_posts/2016-12-05-last-week-in-manageiq-make-hawkular-tenant-changeable.html.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Last Week in ManageIQ: Last Week in ManageIQ: Make Hawkular tenant changeable"
+author: jvlcek
+date: 2016-12-02 11:50:57 -05:00
+tags: LWIMIQ
+comments: true
+published: true
+---
+
+Howdy! I am [JoeV](https://github.com/jvlcek). I work with the ManageIQ *Platform Team*.
+Below is a review of the changes made *Last Week in ManageIQ* land.
+
+## Featured
+
+As we continue to stabilize the code base for the upcoming Euwe release
+52 contributors posted a hefty [97 PRs into master][PRs merged last week] and [329 commits][Commits merged last week]
+with a *tiebreaker* speed that would garner the admiration of
+[Magnus Carlsen](http://www.nytimes.com/topic/person/magnus-carlsen?inline=nyt-per)
+
+If you would like to join the rodeo and help contribute to moving the stampede, you can choose from our list of [open issues]. 
+
+## Improved
+
+Many improvements were made this past week, a few of which are:
+
+Multiple improvements were made to our _**API**_ by [Alon Goldboim](https://github.com/alongoldboim)
+
+  - ### https://github.com/ManageIQ/manageiq/pull/12838
```

Comment:
@jvlcek - The Markdown syntax wasn't able to apply correctly on a number of the urls - I believe it is because of the `-` before the `###` - The page is rendering with the `###` as part of the url.